src/components/TransferInfoCard.js

```diff
@@ -68,7 +71,12 @@ const TransferInfoCard = ({
               <Typography className={classes.cardHeaderContent} component="div">
                 <CirclesLogoSVG height={fontSize} width={fontSize} />
                 <span>{text}</span>
-                {isLoading && <CircularProgress size={fontSize} />}
+                {isLoading && (
+                  <CircularProgress
+                    classes={{ svg: classes.svg }}
```

Comment:
Have you checked if other Circular Progress are purple or grey? 
This feels like an adhoc way to fix the problem. I think all Circular progress should be purple not just this one. Do we know why the color has changed in this upgrade?

Reply:
correct